Recruit-Training/Basic-Recruit-Training/person/person.cpp

```diff
@@ -0,0 +1,70 @@
+#include <iostream>
+#include <sstream>
+#include <string>
+
+#include "Person.h"
+using namespace std;
+
+Person::Person()
+{
+	
+}
+
+Person::~Person()
+{
+	
+}
+
+Person::Person(const std::string& name, int age)
+{
+	age_ = new int;
+	*age_ = age;
+	name_ = name;
+}
+
+int Person::getAge() const
+{
+	return *age_;
+}
+
+const string& Person::getName() const
+{
+	return name_;
+}
+
+void Person::setName(const std::string& newName)
+{
+	name_ = newName;
+}
+
+void Person::setAge(int newAge)
+{
+	*age_ = newAge;
+}
+
+int Person::combinedAge(Person** personArray, int size)
+{
+	int i;
+	int sum_age;
+	sum_age = 0;
+	
+	for(i = 0; i < size; i++)
+	{
+		sum_age = sum_age + personArray[i][0].getAge();
```

Comment:
Using `[0]` is correct, but personArray[i] doesn't really return a pointer to an array. It just returns a pointer to a singular object. I would be better to dereference it instead.